templates/encrypt.nginx.tmpl

```diff
@@ -10,11 +10,27 @@ server {
     listen [::]:443 ssl http2 default_server;
     ssl_certificate {{fullchain}};
     ssl_certificate_key {{privkey}};
+    ssl_dhparam {{dhparam}};
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
```

Comment:
These settings result in an A+ rating for the ssl connection ([test](https://www.ssllabs.com/ssltest/)). I'll look to add extra info.